### ISSUE TITLE
no-unused-properties: Misses v-bind inside <style>

### ISSUE THREAD

**Issue by ilyub** [vuejs/eslint-plugin-vue#1710]
**Tell us about your environment**

- **ESLint version: 8.2.0
- **eslint-plugin-vue version: 8.0.3

**What did you do?**	

```vue
<script lang="ts">
export default defineComponent({
  props: {
    iconHeight: { required: true, type: String }
  }
}
</script>


<style scoped lang="scss">
.icon {
  height: v-bind(iconHeight);
}
</style>
```

**What did you expect to happen?**

No warning, because I use iconHeight inside style.

My use of v-bind is described in this Vue docs:
https://v3.vuejs.org/api/sfc-style.html#state-driven-dynamic-css

**What actually happened?**

Getting error:
'iconHeight' of property found, but never used.

**Repository to reproduce this issue**

I think the issue is clear without repository.

This rule is not looking for the property insde v-bind in <style> section.

**Comment by ota-meshi** [comment]
I can't reproduce the problem. I close this issue as your repository is not provided to reproduce the issue.
